geoportailv3/static/js/appmodule.js

```diff
@@ -7,12 +7,18 @@ goog.provide('app');
 
 goog.require('ngeo');
 goog.require('ngeo.MockLocationProvider');
+goog.require('ol.has');
 
 
 /**
  * @type {!angular.Module}
  */
-app.module = angular.module('app', [ngeoModule.name, 'gettext', 'ngAnimate']);
+app.module = angular.module('app', [ngeoModule.name, 'gettext', 'ngAnimate'])
+    .run(function() {
+      if (!ol.has.TOUCH) {
+        goog.dom.classlist.add(document.body, 'no-touch');
```

Comment:
`goog.require('goog.dom')` (I notice that we have a lot of missing `goog.require`'s. It's not a problem until it is a problem.)